### ISSUE TITLE
perf(model): reuse and clear Q8 prefill attention-output panel

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#11594]
Parent #11580. Tier 2 serving / Core. For native Q8 prefill; problem: prefillBatchedQ allocates P*nH*hd output floats per layer. Today each layer allocates a zeroed panel; better: one request-local panel cleared before each attention call. Scope exactly internal/model/quant_forward.go and internal/model/q8_attn_panel_reuse_test.go. No scheduling, ABI, SIMD, engine, quality or override changes. Source invariants: attnPrefillInto joins workers before return; quantizeBatchPanelInto completes parFor; projection GEMM consumes its panel synchronously (including blocking Accelerate call). Output accumulation requires clear on every layer. Separate test/red role must first capture exact native hidden/logits/KV parity against current allocation path and failing allocation regression; implementation role may edit production file only. One targeted Go test command covers serial/two-worker initial+append, conservative byte savings, and clear-required parity. Preserve artifacts in allocated root scratch; no throughput claims. Dedupe open/closed searches attention-output reuse and Q8 attention-output panel found no matching unit.

**Comment by anthony-chaudhary** [comment]
Authorized separate test/red role completed via guarded Codex in managed test-q8-panel-11594 checkout with live exclusive DOS lease supervised by its actual acquiring process. Initial /var vs /private/var identity refusal resolved by canonical path; no guard edits/bypass. Red: six exact native CPU parity cases, allocation saves -2..5 B/op against 4147 floor, FAIL16.390s. Implementation adds one request-local attnOut allocation and clear each layer only. Independent-role test transferred byte-identically (SHA256 7a5f9fad6539689099b51a158c359412ca4e1573d1d0eb6738a985981b9a598d). go test ./internal/model -run ^TestQ8AttnPanelReuseExactAndAllocation$ -count=5 -v PASS109.041s, 30 exact hidden/logits/KV cases; savings4606..4627 B/op. Scope fak validate build/vet/gofmt/targeted test PASS107.794s. Candidate uncommitted in managed model-13f78013801b; root allocated scratch issue-11594-recovery preserves both files, hashes, raw red/green logs and validation. Awaiting coordinator candidate witness; no throughput claim.

**Comment by anthony-chaudhary** [comment]
Resolved by `ee375a2` (resolving commit). Closed by the DOS dispatch loop's close-resolved arm, witnessed via `dos commit-audit` (verdict OK / diff-witnessed). Reopen if this does not fully resolve it.
